core/tests/firebase_admin_sdk_stub_test.py

```
@@ -0,0 +1,206 @@
+
+class FirebaseAdminSdkStubTests(test_utils.TestBase):
+    """Testes para o Firebase Admin SDK Stub."""
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.stub = firebase_admin_sdk_stub.FirebaseAdminSdkStub()
+        self.mock_request = webapp2.Request.blank('/')
+        self.mock_response = webapp2.Response()
+
+    def test_establish_auth_session_sets_active_state(self) -> None:
+        self.assertFalse(getattr(self.stub, '_is_session_active'))
```

Ruff (0.15.14)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)